descope/tests/mocks/mgmt/managementmock.go

```diff
@@ -900,6 +900,10 @@ func (m *MockProject) Clone(name string, tag descope.ProjectTag) (*descope.NewPr
 	return m.CloneResponse, m.CloneError
 }
 
+func (m *MockProject) Delete() error {
+	return nil
```

Comment:
it will be nice to add a configuration for mockProject error

noting that this is an external package that may be use by consumer

Reply:
@omercnet what about this one? may help Descoper customer to test

Reply:
done